### PR TITLE
Updated link for button and image

### PR DESCRIPTION
**Pull request by tjrexer**
## Related Issue

#719 

## Your solution
Replaced the Link components with a.

## How Has This Been Tested?
Manually tested in local environment.

### CODE REVIEW

**Comment by Pavel910** [comment]
Just rechecked a PR that was submitted not so long ago, related to this issue, but in a Slate link.
https://github.com/webiny/webiny-js/pull/688/files

So what I want to achieve is use those changes from that PR, and move them down to `@webiny/react-router`, into its `Link` component and do that check there: https://github.com/webiny/webiny-js/blob/master/packages/react-router/src/Link.tsx#L14

That way we can continue using that Link component everywhere and it will take care of internal/external links.

**Comment by tjrexer** [comment]
Makes sense, I was trying more for a hotfix since this is breaking production right now.

**Comment by tjrexer** [comment]
Updated based on your recommendations.  Got it working for the site app, appears to be an issue with app-admin package not having all of the props.  If you guys know of a quick way to resolve this great, otherwise I'll get to it when I can.

File: packages/react-router/src/Link.tsx [hunk 1]
```diff
@@ -1,17 +1,36 @@
 import React, { useContext, useEffect } from "react";
-import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
+import { Link as RouterLink } from "react-router-dom";
 import { RouterContext } from "./context/RouterContext";
+import PropTypes from "prop-types";
 
-export type LinkProps = RouterLinkProps;
```
[the remark this answers is not in the document]
Comment (tjrexer) [reply packages/react-router/src/Link.tsx]: Did it to control the props better, ended up over-complicating the problem.  Pulled that all back and used default.

File: packages/react-router/src/Link.tsx [hunk 2]
```diff
@@ -1,17 +1,36 @@
...
 
-function Link({ children, ...props }: LinkProps) {
-    const { onLink } = useContext(RouterContext);
+function Link({children, ...props}: LinkProps) {
+        const { onLink } = useContext(RouterContext)
 
-    useEffect(() => {
-        onLink(props.to as string);
-    }, [props.to]);
+        const isInternal = 'href' in props ? props.href.startsWith("/") : false
```
[the remark this answers is not in the document]
Comment (tjrexer) [reply packages/react-router/src/Link.tsx]: I did have to change up the logic a little bit.  Assuming that if it isn't a string that is passed in, then it is a LocationObject which means it's internal.  pulled out the extra rel & newTab logic and referred back to the default react-router-dom props.

File: packages/react-router/src/Link.tsx [hunk 3]
```diff
@@ -1,17 +1,30 @@
 import React, { useContext, useEffect } from "react";
 import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
 import { RouterContext } from "./context/RouterContext";
+import PropTypes from "prop-types";
```
Comment (Pavel910) [inline packages/react-router/src/Link.tsx]: I think we're safe to remove this, the props are checked with TS.

Comment (tjrexer) [reply packages/react-router/src/Link.tsx]: Yea, my bad.  Had it removed, then forgot to clean it up after I undid something else I was working on.

**Comment by Pavel910** [comment]
Just to let you know that this is published to `npm` :)

**Pull request merged**